storehaus-mongodb/src/main/scala/com/twitter/storehaus/mongodb/MongoStore.scala

```diff
@@ -93,24 +96,24 @@ class MongoStore[K: MongoValue, V: MongoValue: Manifest] (
 
   protected def getValue(valueObject: MongoDBObject): Option[V] = {
```

Comment:
I think it `.isEmpty` is fine if we are never looking at the `Some(contents)` to get the data out.